### PR TITLE
brautify and perfect the pages

### PR DESCRIPTION
**Pull request by ericqiangguo**
hi guys, according the the issue #17, I have made some improvements about the layout of web pages. Please give me some comments.
thanks!

### CODE REVIEW

**Review by xingweitian — CHANGES_REQUESTED** [review]
Thanks for your contribution! Please prevent duplicating files and re-request us to review.

**Review by zhangjiangqige — CHANGES_REQUESTED** [review]
All files should still be stored in the root folder rather than in a new folder ("kitchen_towel_beautify").

You can view the changes on the web page of creating a pull request and check if the changes in the PR are really correct.

**Comment by ericqiangguo** [comment]
> All files should still be stored in the root folder rather than in a new folder ("kitchen_towel_beautify").
> 
> You can view the changes on the web page of creating a pull request and check if the changes in the PR are really correct.

thanks for you comments. I already changed the level of the documents.

File: addroom.html [hunk 1]
```diff
@@ -17,52 +18,126 @@
...
+  
+  
+  
   <body>
-    <div id="map"></div>
+      <ul>
+          <li><img  src="logo.png" width="40%" height="40%"><li>
+          <li><a class="" href="#home">Home</a></li>
+          <li><a href="file:///Users/ericguochn/Downloads/kitchen-towel-master%202/signup.html#news">Sing Up</a></li>
+          <li><a href="file:///Users/ericguochn/Downloads/kitchen-towel-master%202/index.html">Sing In</a></li>
```
Comment (zhangjiangqige) [inline addroom.html]: Hyperlinks should be relative to the domain name, such as "/index.html", not the path on your own computer.

Also it should be "Sign up" and "Sign in", not "Sing".

File: index.html [hunk 2]
```diff
@@ -13,42 +13,135 @@
     />
     <meta name="author" content="LayoutIt!" />
 
+    <style type="text/css">
+
+      /*set the background*/
+      .bg {
+            background: url('https://jlprojects.com/wp-content/uploads/2017/06/Sea-Ranch-Boca-Raton-Condo-Remodeling-3.jpg');
```
Comment (zhangjiangqige) [inline index.html]: This image would be better downloaded and put in a folder named "img", and use the link "/img/someimg.jpg". Some websites forbid external links.

File: index.html [hunk 3]
```diff
@@ -13,42 +13,135 @@
...
   </head>
+  
+ 
+  
   <body>
+      <ul>
+          <li><img  src="logo.png" width="40%" height="40%"><li>
+          <li><a class="" href="#home">Home</a></li>
+          <li><a href="file:///Users/ericguochn/Downloads/kitchen-towel-master%202/signup.html#news">Sing Up</a></li>
+          <li><a href="">Sing Out</a></li>
```
Comment (zhangjiangqige) [inline index.html]: Hyperlink and wrong spelling.

File: signup.html [hunk 4]
```diff
@@ -18,65 +18,170 @@
...
-    <link href="css/bootstrap.min.css" rel="stylesheet" />
-    <link href="css/style.css" rel="stylesheet" />
+    
+    
+    
+    <style type="text/css">
+     /*set the background*/
+
+      .bg {
+            background: url('https://wallpapercave.com/wp/wp2163304.jpg');
```
Comment (zhangjiangqige) [inline signup.html]: Download and "img" folder, same as above.

File: signup.html [hunk 5]
```diff
@@ -18,65 +18,170 @@
...
   </head>
+
   <body>
+
+
+      <ul>
+          <li><img  src="logo.png" width="40%" height="40%"><li>
+          <li><a class="" href="#home">Home</a></li>
+          <li><a href="file:///Users/ericguochn/Downloads/kitchen-towel-master%202/index.html">Sing In</a></li>
+          <li><a href="">Sing Out</a></li>
```
Comment (zhangjiangqige) [inline signup.html]: Hyperlink and spelling.

**Review by zhangjiangqige — CHANGES_REQUESTED** [review]
Please see comments, and `.gitignore` and `README.md` shouldn't be removed.

File: addroom.html [hunk 6]
```diff
@@ -78,9 +78,9 @@
   <body>
       <ul>
           <li><img  src="logo.png" width="40%" height="40%"><li>
-          <li><a class="" href="#home">Home</a></li>
-          <li><a href="file:///Users/ericguochn/Downloads/kitchen-towel-master%202/signup.html#news">Sing Up</a></li>
-          <li><a href="file:///Users/ericguochn/Downloads/kitchen-towel-master%202/index.html">Sing In</a></li>
+          <li><a class="" href="">Home</a></li>
+          <li><a href="signup.html">Sign Up</a></li>
+          <li><a href="index.html">Sign In</a></li>
           <li><a href="">Sing Out</a></li>
```
Comment (zhangjiangqige) [inline addroom.html]: Sign out

File: index.html [hunk 7]
```diff
@@ -17,7 +17,7 @@
 
       /*set the background*/
       .bg {
-            background: url('https://jlprojects.com/wp-content/uploads/2017/06/Sea-Ranch-Boca-Raton-Condo-Remodeling-3.jpg');
+            background: url('index_img.jpg');
```
Comment (zhangjiangqige) [inline index.html]: Please put all images in a new folder named "img", just like all javascripts are in the folder "js" and css files are in "css".

File: signup.html [hunk 8]
```diff
@@ -101,8 +101,8 @@
 
       <ul>
           <li><img  src="logo.png" width="40%" height="40%"><li>
-          <li><a class="" href="#home">Home</a></li>
-          <li><a href="file:///Users/ericguochn/Downloads/kitchen-towel-master%202/index.html">Sing In</a></li>
+          <li><a class="" href="">Home</a></li>
+          <li><a href="index.html">Sing In</a></li>
```
Comment (zhangjiangqige) [inline signup.html]: Sign in

**Review by zhangjiangqige — CHANGES_REQUESTED** [review]
Please see comments, and restore `.gitignore`.

Files whose names start with a dot are hidden files on macOS and can't be seen by default in Finder, so in order to see them you can use the terminal and run `ls -a` which lists all files. Using an editor like VS code to open the project folder directly is also a good choice; it displays all files by default.

**Pull request merged**